charts/prometheus-redis-exporter/templates/deployment.yaml

```diff
@@ -36,6 +36,10 @@ spec:
           {{- range $key, $value := .Values.extraArgs }}
             - --{{ $key }}={{ $value }}
           {{- end }}
+          securityContext:
+          {{- range $key, $value := .Values.securityContext }}
+            {{ $key }}: {{ $value }}
+          {{- end }}
```

Comment:
what if securityContext in values.yaml is empty?

Reply:
In that case, we will see empty securityContext in the manifest, and container will run under the root user by default.
I could add to the values.yaml something like this
`securityContext: {
    runAsUser: 0,
    runAsGroup: 0
}`
or this
`securityContext: {}`